src/main.rs

```diff
@@ -160,7 +155,6 @@ fn main() {
         );
     } else {
         let filter_handlebars_cc = bin_dir.join("filter.cc.handlebars");
-        output_name.set_extension("cc");
```

Comment:
Something went wrong here. This is incorrect.

Reply:
Fixed